Source/Meadow.Contracts/Hardware/Networking/ICellNetworkAdapter.cs

```diff
@@ -20,8 +21,29 @@ public interface ICellNetworkAdapter : INetworkAdapter
     string AtCmdsOutput { get; }
 
     /// <summary>
-    /// Scans for networks the Adapter detects
+    /// Performs an offline scan for networks detected by the adapter
     /// </summary>
-    /// <returns>A list of CellNetworks</returns>
-    CellNetwork[] Scan();
-}
+    /// <returns>An array of CellNetwork objects representing available networks.</returns>
+    CellNetwork[] OfflineNetworkScan();
+
+    /// <summary>
+    /// Get current signal quality
+    /// </summary>
+    /// <param name="timeout"></param>
+    /// <return>Cell signal quality</return>
+    double GetSignalQuality(int timeout = 30);
```

Comment:
I'm not sure if we should let the default values as we did. If someone has any suggestions, please let us know.